test/run_cargo_test.sh

```diff
@@ -2,8 +2,14 @@
 
 set -eou pipefail
 
+source ./support/ci/shared.sh
+
+: "${toolchain:=stable}"
+
 while [[ $# -gt 1 ]]; do
   case $1 in
+    --nightly )             toolchain=$(get_current_toolchain)
```

Comment:
It's probably time to change the name of `get_current_toolchain` to something like `get_nightly_toolchain` or `get_current_nightly_toolchain`.